models/palm_detection_mediapipe/demo.cpp

```diff
@@ -0,0 +1,2377 @@
+#include <chrono>
+#include <cmath>
+#include <opencv2/opencv.hpp>
+#include <string>
+#include <vector>
+
+const std::vector<std::pair<cv::dnn::Backend, cv::dnn::Target>>
+    backend_target_pairs = {
+        {cv::dnn::DNN_BACKEND_OPENCV, cv::dnn::DNN_TARGET_CPU},
+        {cv::dnn::DNN_BACKEND_CUDA, cv::dnn::DNN_TARGET_CUDA},
+        {cv::dnn::DNN_BACKEND_CUDA, cv::dnn::DNN_TARGET_CUDA_FP16},
+        {cv::dnn::DNN_BACKEND_TIMVX, cv::dnn::DNN_TARGET_NPU},
+        {cv::dnn::DNN_BACKEND_CANN, cv::dnn::DNN_TARGET_NPU}};
+
+class MPPalmDet {
+   private:
+    std::string model_path;
+    float nms_threshold;
+    float score_threshold;
+    int topK;
+    int backend_id;
+    int target_id;
+    cv::Size input_size;
+    cv::dnn::Net model;
+    std::vector<cv::Point2f> anchors;
+
+    std::vector<cv::Point2f> loadAnchors() {
```

Comment:
If possible, could you pupt this multi-thousand lines to the end of this file for better readibility?